### ISSUE TITLE
Does not work with Jest

### ISSUE THREAD

**Issue by muhmud** [apigee-127/bagpipes#31]
Hi,

There's a bug fix for release 0.1.0 in bagpipes/lib/fittingTypes/user.js when using bagpipes with Jest, however, this hasn't been released?

Would that be possible?

Thanks,

**Comment by WaldoJeffers** [comment]
Yes @theganyo please; this is absolutely critical for my team. It's basically impossible to require the file where swagger-node is instanciated (usually `app.js`), making it impossible to test with Jest or other test frameworks.

**Comment by theganyo** [comment]
Done! 0.1.1 has been released. Hope this works for you!

**Comment by WaldoJeffers** [comment]
@theganyo works like a charm, thank you!

**Comment by muhmud** [comment]
Works for me too, thanks!

**Comment by theganyo** [comment]
@WaldoJeffers @muhmud The patch submitted for this had to be rolled back in 0.1.2 because of regressions. Are you saying that it still works fine or that you're pinning on version 0.1.1?

**Comment by WaldoJeffers** [comment]
@theganyo Haven't tried version 0.1.2. To fix my issue, I had to lock my version to 0.1.1 (which was 0.1.0 at the time). Does that help @theganyo ?

**Comment by theganyo** [comment]
I was just curious and wanted to give you the heads up as I assume it will no longer work. A better considered and tested PR will be needed before this goes in again.

**Comment by WaldoJeffers** [comment]
Okay, thanks. The version is locked down in my `yarn.lock` file, but thanks for the heads up. Appreciated. Should we re-open this issue or not?

**Comment by theganyo** [comment]
Sure. But the issue title is incorrect. I'll reopen and change it.
